packs/spell-effects/spell-effect-fire-shield.json

```
@@ -98,19 +98,21 @@
                     "field": "item|system.level.value"
                 }
             },
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "label": "Defending against a water effect",
```

```suggestion
                "label": "PF2E.SpecificRule.FireShield.DefendingAgainstWaterLabel",
```

Localize this. If you haven't localized something before go to `static\lang\re-en.json` and find the SpecificRule section. Insert `FireShield` as a new object where it should be alphabetically then make the subobject `"DefendingAgainstWaterLabel": "Defending against a water effect"`. Then reference it in this rule like the suggestion above.